docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
```

Comment:
logs or build output? Some users might want to download the logs and grok them manually, others might want to just look at the build output on a webpage.

Reply:
Good point. I think both? some of the failure data is in the build logs, but others might be inside an output/artifact (ex. logcat)